gcs/src/components/navbar.jsx

```diff
@@ -117,7 +144,7 @@ export default function Navbar({ currentPage }) {
     socket.on('connected_to_drone', (data) => {
       setAircraftType(data.aircraft_type)
       if (data.aircraft_type != 1 && data.aircraft_type != 2) {
-        showErrorNotification("Aircraft not of type quadcopter or plane")
+        showErrorNotification('Aircraft not of type quadcopter or plane')
```

Comment:
double quotes = string, single quotes = char. Why has this been changed?

Reply:
Strings have always been single and double quotes, single quote's doesn't mean char in JS. The prettier config, set 9 months ago indicate that strings should be single quotes, it's easier to type out single quotes and easier to understand